config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml

```diff
@@ -386,6 +386,7 @@ presubmits:
         - --env=KUBE_PROXY_DAEMONSET=true
         - --env=ENABLE_POD_PRIORITY=true
         - --env=ENABLE_APISERVER_TRACING=true
+        - --env=KUBE_WATCHLIST_INCONSISTENCY_DETECTOR=true
```

Comment:
This will not work - this will enable the detector for the "test binary" itself, and we want to enable it for the actual components...

You need to roughly duplicate the logic from mutation detector...

Reply:
Could you please give me some pointers ? 

This repository seems to be setting `ENABLE_CACHE_MUTATION_DETECTOR` only.

Reply:
Look for `ENABLE_CACHE_MUTATION_DETECTOR` in k/k and see how it works.

Reply:
I found https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/gci/configure-helper.sh#L2216 
which seems to be setting `ENABLE_CACHE_MUTATION_DETECTOR` for KCM.